sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1293,7 +1293,7 @@ object SQLConf {
       .createWithDefault(true)
 
   val STREAMING_METRICS_ENABLED =
-    buildConf("spark.sql.streaming.metricsEnabled")
```

Comment:
If you want to change this existing naming space, you should change all others consistently, e.g. `spark.sql.streaming.fileSource.schema.forceNullable`. `sql.streaming` looks pretty clear to me.

You should also consider moving from `SQLConf` to `SparkConf` if you want to change the name space consistently. (e.g., spark.pyspark vs spark.sql...pyspark)

It doesn't look like worth changing to me given the overhead.

Reply:
I agree that changing `spark.sql.streaming.metricsEnabled` appears to open a can of worms. Maybe we should just stick to the other 2 parameters that are easier to change.